.travis.yml

```diff
@@ -24,6 +35,7 @@ deploy:
     - dist/latest-mac.yml
```

Comment:
When you run `yarn build_electron` does that produce `dist/github/latest-linux.json` & `dist/latest-linux.yml` for you? If so, we may want to include them for auto-update support

Reply:
No,
I only get this:

```bash
$ ls dist/
linux-unpacked  partyshare.busterlabs.xyz-1.2.7-linux.deb  partyshare.busterlabs.xyz-1.2.7-linux.tar.gz
```

I'll check how electron deals with Linux builds. Have to change the file reference too :-)

Reply:
Ah, no worries. Auto-update might not be supported on linux, I'll check.

Reply:
ok, you're right. unfortunately. I noticed my build crashed when trying to update. Can/should we disable asar for linux builds?

Reply:
Hmm, it looks like we should be creating a `latest-linux.*` file [according to the docs](https://www.electron.build/auto-update#file-generated-and-uploaded-in-addition)

Reply:
locally I don't get one. maybe do a `ls -l` at `before_deploy` key in `.travis.yml`, so we can see what is actually there.

Reply:
Regarding the "crash":

$ ipfs daemon; partyshare.busterlabs.xyz

My log output: https://gist.github.com/hexagon6/14303b59b269877eafcd384b7dc17909

It crashes with `ipfs version 0.3.10`, probably I should update.

Reply:
Nope, even with version 0.4.13 I get this Error:

```
_getConfig Error: non-zero exit code 1

            while running: /opt/Partyshare/resources/app.asar.unpacked/node_modules/go-ipfs-dep/go-ipfs/ipfs config show


            Error: api not running

    at Stream.listeners.done.once (/opt/Partyshare/resources/app.asar/node_modules/ipfsd-ctl/src/exec.js:39:11)
    at Stream.f (/opt/Partyshare/resources/app.asar/node_modules/once/once.js:25:25)
    at emitTwo (events.js:125:13)
    at Stream.emit (events.js:213:7)
    at emitTwo (events.js:125:13)
    at ChildProcess.emit (events.js:213:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
```